feature-libs/asm/components/customer-emulation/customer-emulation.component.ts

```diff
@@ -2,7 +2,6 @@ import { Component, OnDestroy, OnInit } from '@angular/core';
 import { User, UserService } from '@spartacus/core';
 import { Observable, Subscription } from 'rxjs';
 import { AsmComponentService } from '../services/asm-component.service';
-
 @Component({
```

Comment:
Can you revert the change to lower the noise in the PR and have a more consistent code formating.